walletlibrary/src/main/java/com/microsoft/walletlibrary/networking/entities/openid4vci/credentialmetadata/CredentialConfiguration.kt

```diff
@@ -18,19 +19,23 @@ internal data class CredentialConfiguration(
     val scope: String? = null,
 
     // The crypto binding methods supported (ex. did:jwk).
-    val cryptographic_binding_methods_supported: List<String>? = null,
+    @SerialName("cryptographic_binding_methods_supported")
```

Comment:
So, isn't there a change in casing for these fields? Is it only for credentialSubject?